listenbrainz/listenstore/timescale_utils.py

```diff
@@ -130,7 +130,7 @@ def refresh_listen_count_aggregate():
            Assuming today is 2022-01-01 and this function is called for year_offset 1 and
            year_count 1 then all of 2021 will be refreshed.
     """
-
+    logger.info("Starting to refresh continuous aggregates:")
     timescale.init_db_connection(config.SQLALCHEMY_TIMESCALE_URI)
```

Comment:
As discussed in chat - we should run this function within an app context which means that we'd already have a logger configured, and a connection to timescale set up